services/core-web/src/HOC/ApplicationGuard.js

```diff
@@ -101,10 +103,14 @@ export const ApplicationGuard = (WrappedComponent) => {
       await Promise.all([
         this.props.fetchImportedNoticeOfWorkApplication(id).then(({ data }) => {
           this.setState({ mineGuid: data.mine_guid });
+          this.props.fetchDraftPermitByNOW(data.mine_guid, id);
           if (data.application_type_code === "NOW") {
             this.props.fetchOriginalNoticeOfWorkApplication(id);
             this.props.fetchImportNoticeOfWorkSubmissionDocumentsJob(id);
           }
+          if (data.application_type_code === "ADA") {
+            this.props.fetchPermits(data.mine_guid);
```

Comment:
Don't both ADA and NOW need the permits?

Reply:
nope, they both require the draft permits and it is done in line 106

Reply:
Is this for the dropdown list of permits on the draft permit modal?

Reply:
if so, NoW amendments need that data as well.

Reply:
Nope, this is not for drop-down, this is for check if the type field should be available for edit. The draft permit modal fetches this data itself on load.